doc/using-demo-app.md

```diff
@@ -39,7 +39,7 @@ This app uses [EdgeX Core Services][edgex-core-services], [EdgeX ONVIF device se
     make install-edge-video-analytics
     ```
 
-1. Run Edge Video Analytics in a dedicated terminal.
+1. Run Edge Video Analytics in a dedicated terminal. (Docker Compose command in the yaml file may need to be changed from docker-compose to docker compose)
```

Comment:
We should probably just make that the default now. Also we should allow overridding via:
```shell
make DOCKER_COMPOSE="docker compose" run-edge-video-analytics
```

but i dont think it was setup that way (using an = instead of ?=).